dev/io.openliberty.io.smallrye.openapi41.jaxrs/bnd.bnd

```diff
@@ -0,0 +1,49 @@
+#*******************************************************************************
+# Copyright (c) 2024, 2025 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/rejar.props
+
+bVersion=1.0
+
+Bundle-SymbolicName: io.openliberty.io.smallrye.openapi41.jaxrs; singleton:=true
+
+WS-TraceGroup: MPOPENAPI
+
+src: src
+
+javac.source: 11
+javac.target: 11
+
+Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"
+
+Include-Resource: \
+    @${repo;io.smallrye:smallrye-open-api-jaxrs;4.0.9}!/META-INF/services/*
+
+Service-Component: \
+    io.smallrye.openapi.runtime.scanner.spi.AnnotationScanner; \
+    implementation:=com.ibm.wsspi.classloading.ResourceProvider; \
+    provide:=com.ibm.wsspi.classloading.ResourceProvider; \
+    configuration-policy:=optional; \
+    properties:="resources=META-INF/services/io.smallrye.openapi.runtime.scanner.spi.AnnotationScanner"
+
+Import-Package: \
+    io.smallrye.openapi.jaxrs,\
+    *
+
+Export-Package: \
+    io.smallrye.openapi.jaxrs;version=4.0;thread-context=true
```

Comment:
Similarly this could be changed to 4.1 now as well or can wait until after you switch to the new version of small rye.